src/components/BadgeOverlay/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+
+import abbreviateNumber from '../AvatarGroup/helpers/AbbreviateNumber';
```

Comment:
we need to move this to `src/libs/utils` in order to reuse and then change its name to getSuffixSI